hepstats/hypotests/hypotests_object.py

```diff
@@ -0,0 +1,227 @@
+from .fitutils.api_check import is_valid_loss, is_valid_fitresult, is_valid_minimizer
+from .fitutils.api_check import is_valid_data, is_valid_pdf
+from .fitutils.utils import get_nevents
+
+
+class HypotestsObject(object):
+
+    def __init__(self, input, minimizer):
+        """Base object in `hepstats.hypotests` to manipulate a loss function and a minimizer.
+
+            Args:
+                input : loss or fit result
+                minimizer : minimizer to use to find the minimum of the loss function
+        """
+
+        if is_valid_fitresult(input):
```

Comment:
I would have to check properly but you probably know by heart: are these 2 `is_valid_` mutually exclusive? Otherwise the second if statement could override the first one. Just checking, as it must be the case ;-).

Reply:
Yes they are mutually exclusive.